Monika After Story/game/script-topics.rpy

```diff
@@ -11546,9 +11605,39 @@ label monika_grad_speech_not_paying_attention:
     return
 
 label monika_grad_speech_ignored_lock:
-    #Second timeout, lock speech
     hide screen mas_background_timed_jump
-    #Set false for modified dialogue in the random
+
+    # Still, we make a double check.
+    m 1dsc "...{w=0.2}{nw}"
+    extend 1esd "[player].{nw}"
+
+    $ _history_list.pop()
+    show screen mas_background_timed_jump(5, "monika_grad_speech_ignored_lock_real")
+    menu:
+        m "...[player].{fast}"
+        "I'm listening!":
+            hide screen mas_background_timed_jump
+
+            m 1wsb "Oh!{w=0.5}{nw}"
+            extend 1hua " sorry for that."
+            m 1hksdla "There have a time when you haven't really listened, so I'm a little sensitive."
+            m 1hksdlb "Ahaha~"
+            m 1eub "Anyways, [player], now that you actually {i}heard{/i} my speech, what do you think?{nw}"
+
+            $ _history_list.pop()
+            menu:
+                m "Anyways, [player], now that you actually {i}heard{/i} my speech, what do you think?{fast}"
```

Comment:
Same here.

Reply:
The reason I did this is because I wanted the menu options here to correspond to the original text -- Monika "took the menu out again".